### DISCUSSION TITLE
Pellematic Smart XS

### DISCUSSION THREAD

**Discussion by BarryKaty1974** [dominikamann/oekofen-pellematic-compact#29] [Q&A]
Hello,

I have a Pellematic Smart XS. Is it also possible for this one to connect with Home Assistant trough Json?
I have tried and everything show as unknown.
When i enter de IP:port/password/all in my browser i see "Wait at least 2500ms during requests". 
When i enter a wrong password i see under.
What i do wrong?

Oekofen JSON Interface   V4.00b   http://www.oekofen.at

usage:
http://touch_ip:JSON_port/password/command

====================================================================================
read multiple values:
====================================================================================
http://192.168.1.37:4321/********/all?

Example: http://192.168.1.37:4321/********/hk1?

    all[?]     displays values of all available components
               optional parameter [?] adds meta data

    error
        read current errors

    log[0..3]
        download csv-logging file

    system[?]
        system global variables

    weather[?]
        current weather data

    forecast[?]
        date|temp|cloud|speed|image|code|unit[|sunrise|sunset] code see https://openweathermap.org/weather-conditions

    power[?]
        powermeter data

    stirling[?]
        stirling data

    hk[1..6][?]
        heating circuit data

    thirdparty[1..20][?]
        third party sensor

    pu[1..3][?]
        accu data

    ww[1..3][?]
        domestic hot water data

    sk[1..6][?]
        solar circuit data

    se[1..3][?]
        solar gain data

    circ[1..3][?]
        circulation pump data

    pe[1..4][?]
        pellematic data

====================================================================================
read single value (use '.' as delimitor):
====================================================================================
http://192.168.1.37:4321/********/id(1..6).variable

Example: http://192.168.1.37:4321/********/hk1.L_roomtemp_act

====================================================================================
set value (use '.' as delimitor):
====================================================================================
http://192.168.1.37:4321/********/id(1..6).variable=value

Example: http://192.168.1.37:4321/********/hk1.mode_auto=0

====================================================================================
alternative useage '_' as delimitor:
====================================================================================
http://192.168.1.37:4321/********/id(1..6)_variable=value

Example: http://192.168.1.37:4321/********/hk1_mode_auto=0

    system
        system global variables

    weather
        current weather data

    forecast
        date|temp|cloud|speed|image|code|unit[|sunrise|sunset] code see https://openweathermap.org/weather-conditions

    power
        powermeter data

    stirling
        stirling data

    hk(1..6)
        heating circuit data

    thirdparty(1..20)
        third party sensor

    pu(1..3)
        accu data

    ww(1..3)
        domestic hot water data

    sk(1..6)
        solar circuit data

    se(1..3)
        solar gain data

    circ(1..3)
        circulation pump data

    pe(1..4)
        pellematic data

only variables without a leading 'L_' can be set.

**Comment by dominikamann** [comment]
Hi @BarryKaty1974 
what refresh time did you configured when you added your Pellematic over my integration?

**Reply by dominikamann** [reply]
And can you share the HA Logfiles please?

**Comment by BarryKaty1974** [comment]
Hello Domini,

After updating to the new version of HA, I had to change something because of errors. To make sure it wasn't that, I first downgraded to the previous version or HA, reinstalled Okofen and it worked immediately. I updated back to the new version of HA and it continued to work. Thx!
